content/post/115-ipfs-and-igalia-collaborate-on-dweb-in-browsers.md

```diff
@@ -0,0 +1,195 @@
+---
+date: 2021-01-15
+url: /115-ipfs-and-igalia-collaborate-on-dweb-in-browsers/
```

Comment:
I believe  the convention for non-weekly articles is to include date in the url slug, so its easy to tell how old content is:

```suggestion
url: /2021-01-13-ipfs-and-igalia-collaborate-on-dweb-in-browsers/
```

Reply:
mmh, there does not seem to be any other posts like that?
FWIW, I followed https://github.com/ipfs/blog#create-a-new-blog-post here.